application/dash_application/views/bcl2fastq.py

```diff
@@ -89,6 +108,27 @@ def layout(qs):
                     ),
                 ]
             ),
+            html.Br(),
+            core.Tabs(id=ids['tabs_data_table'], children=[
+                core.Tab(label='Known Barcodes', children=[
+                    dash_table.DataTable(
+                        id=ids["known_data_table"],
+                        columns=KNOWN_DATA_TABLE_COLS,
+                        export_format='csv',
+                        export_headers='names',
+                        sort_action='native',
```

Comment:
For some reason, I'm unable to copy-paste from these tables. I had hoped that adding `include_headers_on_copy_paste=True,` might fix it, but it doesn't seem to. This is merely me expressing sadness rather than requesting you fix Dash, though.

Reply:
Yeah, weird bug. The solution is to first Shift-select the text and then copy and paste:

![Peek 2020-02-05 13-15](https://user-images.githubusercontent.com/37381890/73870319-e1ecdb80-4819-11ea-9537-318ecd541eee.gif)

Reply:
:crying_cat_face: that's impressively bad.